### PR TITLE
Update content and artwork for security terms.

### PR DESCRIPTION
**Pull request by LouisBerner**

### CODE REVIEW

**Comment by Jak-MS** [comment]
@alkohli 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-stack/hci/manage/manage-secure-baseline.md [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
 [!INCLUDE [hci-applies-to-23h2](../../includes/hci-applies-to-23h2.md)]
 
-This article describes how to manage baseline security settings for your Azure Stack HCI cluster and the associated drift control mechanism to ensure that the device starts in a known good state.
+This article describes how to manage default security settings for your Azure Stack HCI cluster. You can also modify drift control and protected security settings defined during deployment so your device starts in a known good state.
 
 [!INCLUDE [important](../../includes/hci-preview.md)]
 
-## Enable, disable drift control
+## View security default settings
+
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/manage/manage-secure-baseline.md:21]: Done > removed.

File: azure-stack/hci/manage/manage-secure-baseline.md [hunk 2]
```diff
@@ -1,23 +1,35 @@
...
+### View security default settings in Azure portal
+
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/manage/manage-secure-baseline.md:23]: Done > promoted to H2.

File: azure-stack/hci/manage/manage-secure-baseline.md [hunk 3]
```diff
@@ -1,23 +1,35 @@
...
+Use security default settings to manage cluster security, drift control, and secured core server settings on your cluster.
+
+:::image type="content" source="media/manage-secure-baseline/manage-secure-baseline.png" alt-text="Screenshot that shows Security defaults page on Azure portal." lightbox="media/manage-secure-baseline/manage-secure-baseline.png":::
+
+### Manage security default settings with PowerShell
+
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/manage/manage-secure-baseline.md:29]: Done > Promoted to H2, revised title.

File: azure-stack/hci/manage/manage-secure-baseline.md [hunk 4]
```diff
@@ -1,23 +1,35 @@
...
+With drift protection enabled, you can only modify non-protected security settings. To modify protected security settings that form the baseline, you must first disable drift protection. To view and download the complete list of security settings, see [SecurityBaseline](https://aka.ms/hci-securitybase).
+
+## Modify security default settings
```
[the remark this answers is not in the document]
Comment (LouisBerner) [reply azure-stack/hci/manage/manage-secure-baseline.md:33]: Done > revised title.

**Comment by LouisBerner** [comment]
If this PR needs approval, or if the commits need to be pulled into the private repo, contact @ManikaDhiman. Thanks.

**Pull request merged**